src/components/scenes/CurrencySettingsScene.js

```diff
@@ -53,7 +53,7 @@ export function CurrencySettingsComponent(props: Props) {
 
   // Are the electrum servers enabled? What are they?
   const defaults = asMaybe(asElectrumDefaults)(defaultSettings)
-  const settings = asMaybe(asElectrumSettings)(userSettings)
+  const settings = asMaybe(asElectrumSettings, { electrumServers: [], disableFetchingServers: false })(userSettings)
```

Comment:
This is an OK fix for now. 
But we need to fix this entire 'CurrencySettingsComponent' to not have the settings hard coded in the GUI and instead receive a `userSettingsType` from the plugin.